src/spec.c

```diff
@@ -171,6 +171,17 @@ const char *IndexSpec_GetFieldNameByBit(const IndexSpec *sp, t_fieldMask id) {
   return NULL;
 }
 
+// Get the field spec by the field mask.
+const FieldSpec *IndexSpec_GetFieldByBit(const IndexSpec *sp, t_fieldMask id) {
```

Comment:
Only for TEXT? If so, name is a bit misleading

Reply:
But I agree in general.